kpi/models/abstract_models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.utils import timezone
+
+
+def get_current_time():
+    """
+    Return the current time so that the timestamp is evaluated at the time of model instantiation rather than
+    at the time of class definition
+    """
+    return timezone.now()
```

Comment:
I would change the name to something like `_get_default_datetime`. 

Notes: The underscore is to indicate to other devs that it should not be outside this file. 

Try to keep the comment under the 80 characters per line and explain why we need to use this function instead of using `timezone.now` directly.